docs/jwst/cube_build/main.rst

```diff
@@ -71,7 +71,7 @@ We use the following terminology to define the spectral range divisions of MIRI:
   longest is covered by Band 4-LONG (aka 4C).
 
    For **NIRSpec** we define a *band* as a single grating-filter combination, e.g. G140M-F070LP. The possible grating/filter
-   combinations for NIRSpec are given in the table below.
+   combinations for NIRSpec are given in 
```

Comment:
Shouldn't really just stop a sentence in mid phrase like this. What's wrong with keeping the wording as "... are given in the following table."

Reply:
made change